Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: lint, test, drone
```

Comment:
It's the first time I see `.PHONY` targets separated with commas (instead of just white spaces), is that accepted as well? 🤔

Reply:
Ups... but yes... it was working 😅